src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -722,7 +722,7 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
                         : ''
                 }`}
             >
-                {(this.state.isRollbackTrigger || this.state.isSelectImageTrigger) && !this.state.showConfigDiffView && (
+                {(this.state.isRollbackTrigger || this.state.isSelectImageTrigger) && !this.state.showConfigDiffView && !(this.renderCDModalHeader()=== 'Pre Deployment' || this.renderCDModalHeader()=== 'Post Deployment') &&(
```

Comment:
use .some method instead of repeating conditions

Reply:
It is saying .some does not exist on ReadOnly and string

Reply:
done